lib/pge_tonearest.h

```diff
@@ -0,0 +1,238 @@
+// This code was taken and adapted from LLVM project:
```

Comment:
Can you remind me the history Of this file? Was it created for vita, or did it get removed and replaced in main (in which case we should replace for vita also)

Reply:
It was created for Vita. For whatever reason, VITASDK doesnt have fegetround fesetround functions so the stock rounding modes are incorrect and cause errors such as

- Backgrounds separating
- Layers moving in and out too far

As far as I know, Vita is the only platform to require it.

Reply:
Interesting, thanks for the reminder! This should probably also be useful on 3DS (though it may have fegetround / fesetround from an SDK update since I started the project).

Reply:
I think, one I should to make is to verify this work on big endian (I need to run my PPC VM at QEMU with Debian and run a small test program)